docs/pywbemcli/cmdlineinterface.rst

```diff
@@ -175,7 +175,7 @@ user):
 
 .. code-block:: text
 
-    pywbemcli> [GENERAL-OPTIONS] [COMMAND-GROUP] COMMAND [COMMAND-OPTIONS] [ARGS]
+    pywbemcli> [GENERAL-OPTIONS] [COMMAND-GROUP] COMMAND [ARGS] [CMD-OPTIONS]
```

Comment:
In the actual help, we no longer have `[COMMAND-GROUP]`in the usage lines, and i think that is the right direction. It should be removed in this file as well (this file is the only one that still contains that). Occurs multiple times. Some words need to be added that explain that `COMMAND` is sometimes directly a command and sometimes a command group that has sub-commands.

Reply:
Fixed.  Removed [COMMAND=GROUP] and defined what command means.